runtime/j9vm/jvm.c

```diff
@@ -5138,7 +5150,9 @@ JVM_Send(jint descriptor, const char* buffer, jint numBytes, jint flags)
 
 	Trc_SC_Send_Entry(descriptor, buffer, numBytes, flags);
 
-	retVal = send(descriptor, buffer, numBytes, flags);
+	do {
+		retVal = send(descriptor, buffer, numBytes, flags);
+	} while ((-1 == retVal) && (EINTR == errno));
```

Comment:
does this compile on Windows: `errno`? https://msdn.microsoft.com/en-us/library/windows/desktop/ms737828(v=vs.85).aspx. you will need an `ifdef` over here.
```
#if defined (WIN32)
	retVal = send(descriptor, buffer, numBytes, flags);
#else /* defined (WIN32) */
	do {
		retVal = send(descriptor, buffer, numBytes, flags);
	} while ((-1 == retVal) && (EINTR == errno));
#endif /* defined (WIN32) */
```

Reply:
This compiled on all platforms. I'll add the ifdef for correctness.